lang/CMakeLists.txt

```diff
@@ -222,6 +223,14 @@ if (NOT WIN32)
 		PROPERTY OUTPUT_NAME sclang)
 endif()
 
+if (SC_ABLETON)
```

Comment:
Could we potentially check for the library being installed on the system and use that?